runtime/doc/lua-guide.txt

```diff
@@ -622,7 +622,7 @@ This is equivalent to the following Lua code:
 Autocommand groups are unique for a given name, so you can reuse them, e.g.,
 in a different file:
 >lua
-    local mygroup = vim.api.nvim_create_augroup('vimrc', { clear = false })
+    local mygroup = vim.api.nvim_create_augroup('vimrc', { clear = true })
```

Comment:
Hmmm, isn't the point of the example that you can reuse existing autocommand groups? I interpret that to mean reuse groups without clearing existing autocommands for that group.

Reply:
Ou! You are right. It make sense, sorry for my ignorance. I am going to revert this commit.

Thank you.